Lecture_18/Homework_18.py

```diff
@@ -0,0 +1,32 @@
+class Bot:
+    def __init__(self, name,):
+        self.name = name
+    def say_name(self, name):
+        return Bot(self.say_name)
```

Comment:
task was to print name here
```suggestion
        print(self.name)
```